stock_packaging_calculator/models/product.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import models
+
+
+class Product(models.Model):
+    _inherit = "product.product"
+
+    def product_qty_by_packaging(self, prod_qty, min_unit=None):
+        """Calculate quantity by packaging.
+
+        :prod_qty: total qty to satisfy.
+        :min_unit: minimal unit of measure as a tuple (qty, name).
+                   Default: to UoM unit.
+        :returns: list of tuple in the form [(qty_per_package, package_name)]
+        """
+        packagings = [(x.qty, x.name) for x in self.packaging_ids]
+        if min_unit is None:
+            # You can pass `False` to skip it.
+            single_unit = self.env.ref("uom.product_uom_unit")
```

Comment:
Shouldn't 1 be the `uom_id` of the product?

Reply:
Oh good catch, if I sold a pair of gloves, then 1 pair is in fact 2 units. I want to display 1 pair in that case

Reply:
but to do this really properly we should have UoM on tha packaging (https://github.com/OCA/product-attribute/pull/486) unless you tell me that by default the pkging qty refers always to the unit one